packages/alfa-rules/src/sia-r111/rule.ts

```diff
@@ -54,11 +62,22 @@ export default Rule.Atomic.of<Page, Element>({
  * @public
  */
 export namespace Outcomes {
-  export const HasSufficientSize = Ok.of(
-    Diagnostic.of("Target has sufficient size"),
-  );
+  export const HasSufficientSize = (box: Rectangle) =>
+    Ok.of(WithBoundingBox.of("Target has sufficient size", box));
+
+  export const HasInsufficientSize = (box: Rectangle) =>
+    Err.of(WithBoundingBox.of("Target has insufficient size", box));
+}
+
+// This predicate is assumed to only be used on elements with bounding boxes which should be guaranteed by applicability
```

Comment:
**Chore:** make this into a real TSdoc string, so that it shows up in IDE's contextual help.